tests/runtime/test_stress_docker_runtime.py

```diff
@@ -26,8 +29,9 @@ def test_stress_docker_runtime(temp_dir, runtime_cls, repeat=1):
         # run stress-ng stress tests for 5 minutes
         # FIXME: this would make Docker daemon die, even though running this
         # command on its own in the same container is fine
-        action = CmdRunAction(command='stress-ng --all 1 -t 5m')
-        action.timeout = 600
+        # reduce to 1m
```

Comment:
As you said, seems this is fixed on main now? Then we can remove the above comments

```suggestion
run stress-ng stress tests for 1 minute
```